### PR TITLE
Allow doc and docx for resume in apply for job

### PR DESCRIPTION
**Pull request by cp-shruti-s**

### CODE REVIEW

File: jobs/jobs.go [hunk 1]
```diff
@@ -24,6 +25,8 @@ const (
 	CHARSET = "utf-8"
 )
 
+var Extension string
```
Comment (cp-sumi-k) [inline jobs/jobs.go:28]: Don't make it capitalize. Use `extension`

File: jobs/jobs.go [hunk 2]
```diff
@@ -191,7 +196,7 @@ func GetEmailTemplate(htmlBody string, title string, attachmentBytes *bytes.Buff
 	msg.SetHeader("Subject", title)
 	msg.SetBody("text/html", htmlBody)
 
-	msg.Attach("resume.pdf", gomail.SetCopyFunc(func(w io.Writer) error {
+	msg.Attach("resume"+Extension, gomail.SetCopyFunc(func(w io.Writer) error {
```
Comment (cp-sumi-k) [inline jobs/jobs.go:199]: I think it's `"resume."+Extension`

File: vue-frontend/src/components/ApplyForCareer.vue [hunk 3]
```diff
@@ -152,7 +152,7 @@
                   type="file"
                   class="form-control custom-file-input"
                   name="resume"
-                  accept="application/pdf"
+                  accept="application/pdf,application/msword,application/vnd.openxmlformats-officedocument.wordprocessingml.document,.doc,.docx"
```
Comment (cp-sumi-k) [inline vue-frontend/src/components/ApplyForCareer.vue:155]: What is this `application/vnd.openxmlformats-officedocument.wordprocessingml.document` format?Is it necessary?

**Review by cp-sumi-k — CHANGES_REQUESTED** [review]

File: jobs/jobs.go (hunk 2 above)
Comment (cp-shruti-s) [reply jobs/jobs.go:199]: No, we used `filepath.Ext(fileHeader.Filename)` so this method return with `dot extension`

File: vue-frontend/src/components/ApplyForCareer.vue (hunk 3 above)
Comment (cp-shruti-s) [reply vue-frontend/src/components/ApplyForCareer.vue:155]: - `application/msword` means `.doc`
- `application/vnd.openxmlformats-officedocument.wordprocessingml.document` means `.docx`

Comment (cp-sumi-k) [reply vue-frontend/src/components/ApplyForCareer.vue:155]: Then why we have taken `.doc,.docx` ?

**Review by cp-sumi-k — APPROVED** [review]

**Pull request merged**